swe_bench/inference/run_agent.py

```diff
@@ -30,8 +30,12 @@ def _prepare(inputs):
     cleaned_user_message = re.sub("<patch>.*?</patch>", "", user_message, flags=re.DOTALL)
 
     issues = re.findall("<issue>(.*?)</issue>", user_message, flags=re.DOTALL)
+    issues = [re.sub(r"#{3,4} Versions.*?(?=#{3,4}|\Z)", "", issues[0], flags=re.DOTALL)]
+    traceback = re.findall(r"#{3,4} Actual Results.*", issues[0], flags=re.DOTALL)
+    issues = traceback if traceback else issues
+    code = re.findall("<code>(.*?)</code>", user_message, flags=re.DOTALL)
```

Comment:
Can you explain the corresponding effects of these changes in the issue body?

Reply:
The purpose is：
1. Extract the actual result from the issue
2. Extract all code from message